enclave_manager/avalon_enclave_manager/enclave_type.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum, unique
+
+
+@unique
+class EnclaveType(Enum):
+    """
+    Enclave Types
+    1 - Singleton enclave
+    2 - Key management enclave
+    3 - Work order processing enclave
+    From EEA spec 5.2.
```

Comment:
Does EEA spec mentions Singleton, KME and WPE ?

Reply:
I will correct this.